### ISSUE TITLE
Missing full labelled data for APIBENCH-C 

### ISSUE THREAD

**Issue by Xin-Zhou-smu** [JohnnyPeng18/APIBench#2]
For the APIBENCH-C, it seems that only the test data provides the Ground truth. While the training data does not. Could the authors provide the full data with labels? Thanks
